lib/sprockets/server.rb

```diff
@@ -23,7 +23,7 @@ def call(env)
       start_time = Time.now.to_f
       time_elapsed = lambda { ((Time.now.to_f - start_time) * 1000).to_i }
 
-      if env['REQUEST_METHOD'] != 'GET'
+      if !['GET', 'HEAD'].include?(env['REQUEST_METHOD'])
```

Comment:
Why? Please explain to me why this is worth killing any more time on?

Can we please stop these micro optimizations? I'm so so tired of this.